flaskphiid/__init__.py

```diff
@@ -16,7 +16,7 @@
 
 # load medlp interface
 from compmed.CompMedServiceInterface import CompMedServiceInterface
-import utils.json_parser_utils as JSONParser
+from utils import json_parser_util as JSONParser
```

Comment:
does this still work?
it looks like you brought the utils folder into the flaskphiid to test it out and I'm assuming this import worked then - but does it when you take it out?

Reply:
i brought the utils folder out but ended up not using it. instead i used a renamed utils installed on the compmed setup as compmed_utils to import json_parser. 

it should still work, i just was testing out if there was a conflict with the utils name on my end

Reply:
gotcha - you updated the ComprehendMedicalInterface codebase as well?